monitoring/perf_context.cc

```diff
@@ -186,8 +204,23 @@ std::string PerfContext::ToString(bool exclude_zero_counters) const {
   PERF_CONTEXT_OUTPUT(env_lock_file_nanos);
   PERF_CONTEXT_OUTPUT(env_unlock_file_nanos);
   PERF_CONTEXT_OUTPUT(env_new_logger_nanos);
+  PERF_CONTEXT_BY_LEVEL_OUTPUT(bloom_filter_useful);
+  PERF_CONTEXT_BY_LEVEL_OUTPUT(bloom_filter_full_positive);
+  PERF_CONTEXT_BY_LEVEL_OUTPUT(bloom_filter_full_true_positive);
   return ss.str();
 #endif
 }
 
+void PerfContext::EnablePerLevelPerfContext(int levels){
+  num_levels = levels;
+  perf_context_by_level = new PerfContextByLevel[levels];
```

Comment:
We should support that people call EnablePerLevelPerfContext() multiple times. So if perf_context_by_level is not null, we shouldn't allocate again.